fluent-package/templates/etc/systemd/fluentd.erb

```diff
@@ -1 +1,5 @@
 TD_AGENT_OPTIONS=""
+# The following environment variable will be modified via package maintainer's migration script
+FLUENT_CONF=/etc/<%= package_dir %>/<%= service_name %>.conf
+FLUENT_PLUGIN=/etc/<%= package_dir %>/plugin
+TD_AGENT_LOG_FILE=/var/log/<%= package_dir %>/<%= service_name %>.log
```

Comment:
Will we change these paths, not only the config file?

Then, I think we need some more fixes for macOS.

https://github.com/fluent/fluent-package-builder/blob/3b37b65a56a5c319db6fcc8c1bbdf99d98e53574/fluent-package/dmg/resources/pkg/postinstall.erb#L8-L21

https://github.com/fluent/fluent-package-builder/blob/3b37b65a56a5c319db6fcc8c1bbdf99d98e53574/fluent-package/templates/fluentd.plist.erb#L13

Reply:
Thanks, about migration fixes are piled up, fixes it in another small PR too.